package.json

```diff
@@ -29,19 +30,15 @@
     "husky": "^7.0.2",
     "lerna": "^3.22.1",
     "lint-staged": "^11.1.2",
-    "prettier": "^2.4.1",
-    "typescript": "4.2.2"
+    "prettier": "^2.4.1"
   },
   "dependencies": {
-    "@segment/analytics-next": "^1.32.1",
-    "@theme-ui/color": "^0.3.1",
     "cypress": "^9.5.4",
     "cypress-localstorage-commands": "^1.7.0",
     "cypress-wait-until": "^1.7.2",
     "jest": "^25.5.1",
     "node-fetch": "^2.6.1",
-    "start-server-and-test": "^1.14.0",
-    "validate-npm-package": "^1.0.5"
```

Comment:
what is validate-npm-package?

Reply:
it was apparently used to validate a package.json file.